src/components/vm/overview/firmware.tsx

```diff
@@ -34,13 +37,24 @@ import { supportsUefiXml, labelForFirmwarePath } from './helpers.jsx';
 
 const _ = cockpit.gettext;
 
-const xmlToState = value => value || 'bios';
-const stateToXml = value => value == 'bios' ? null : value;
+const xmlToState = (value: optString) => value || 'bios';
+const stateToXml = (value: string) => value == 'bios' ? null : value;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2185-af23a7d6-20250716-112918-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)